src/explore-education-statistics-common/src/modules/charts/util/createDataSetCategories.ts

```diff
@@ -308,19 +308,27 @@ function getSortedDataSetCategoryRange(
  *
  * 5. Sort and filter out any invalid or empty categories.
  */
-export default function createDataSetCategories(
-  axisConfiguration: AxisConfiguration,
-  results: TableDataResult[],
-  meta: FullTableMeta,
+interface Options {
+  axisConfiguration: AxisConfiguration;
+  data: TableDataResult[];
+  meta: FullTableMeta;
+  includeNonNumericData?: boolean;
+  returnSymbols?: boolean;
+}
```

Comment:
This should go above the docblock for the function and have a line break after it